management/src/test/java/com/kaazing/gateway/management/test/util/TlsTestUtil.java

```diff
@@ -0,0 +1,45 @@
+package com.kaazing.gateway.management.test.util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.security.KeyStore;
+
+public class TlsTestUtil {
+    static char[] password = "ab987c".toCharArray();
+    static String keyStoreFileLocation = "target/truststore/keystore.db";
+
+    public static char[] password() {
+        return password;
```

Comment:
@sanjay-saxena why did you want a clone here?  Wasn't following reason, happy to make change (We will have to make the same change elsewhere)